keps/sig-apps/3335-statefulset-slice/README.md

```diff
@@ -914,8 +978,13 @@ For each of them, fill in the following information by copying the below templat
     - Testing: Are there any tests for failure mode? If not, describe why.
 -->
 
+No other failure modes are known.
+
 ###### What steps should be taken if SLOs are not being met to determine the problem?
 
+If the StatefulSet SLOs are not met, the kube-controller-manager should be
+restarted or examined/debugged.
```

Comment:
What kind of steps should be performed to identify that ordering is misbehaving and how admin could disable the feature?

Reply:
Added some details about checking if the correct ordinal numbers are running, and linked to the rollback section for disabling the feature.